boot/bootutil/include/bootutil/crypto/ecdh_p256.h

```diff
@@ -12,10 +12,18 @@
 
 #include "mcuboot_config/mcuboot_config.h"
 
-#if (defined(MCUBOOT_USE_TINYCRYPT)) != 1
-    #error "One crypto backend must be defined: TINYCRYPT"
+#if (defined(MCUBOOT_USE_MBED_TLS) + \
+     defined(MCUBOOT_USE_TINYCRYPT)) != 1
+    #error "One crypto backend must be defined: either MBED_TLS or TINYCRYPT"
 #endif
 
+#if defined(MCUBOOT_USE_MBED_TLS)
+    #include <mbedtls/ecp.h>
+    #include <mbedtls/ecdh.h>
+
+#    define EC_TAG_INDEX        (65)
```

Comment:
I believe this symbol is not defined for Tinycrypt anymore?

Reply:
fixed this